release/scripts/PowerShell/FhirServerRelease/Private/Set-FhirServerApiUsers.ps1

```diff
@@ -76,6 +76,9 @@ function Set-FhirServerApiUsers {
             $aadUser = New-AzureADUser -DisplayName $userId -PasswordProfile $PasswordProfile -UserPrincipalName $userUpn -AccountEnabled $true -MailNickName $userId
         }
 
+        $upnSecureString = ConvertTo-SecureString -string $userUpn -AsPlainText -Force
+        Set-AzureKeyVaultSecret -VaultName $KeyVaultName -Name "user_$($userId)_id" -SecretValue $upnSecureString | Out-Null
+        Set-AzureKeyVaultSecret -VaultName $KeyVaultName -Name "user_$($userId)_secret" -SecretValue $passwordSecureString | Out-Null
```

Comment:
do we still need the entry below since it's the same as this one?

Reply:
I left it there for now for supporting the existing build, but no, we shouldn't by the time this PR is merged.